### PR TITLE
Fix gosec issue

### PR DESCRIPTION
**Pull request by Prateeknandle**

### CODE REVIEW

**Comment by Prateeknandle** [comment]
@rksharma95 please review the pr

File: src/cluster/clusterMgmtHandler.go [hunk 1]
```diff
@@ -89,7 +89,11 @@ func getResponseBytes(method string, url string, data map[string]interface{}) []
 		return nil
 	}
 	dumpHttpClient(nil, resp)
-	defer resp.Body.Close()
+	defer func() {
+		if err := resp.Body.Close(); err != nil {
+			log.Error().Msgf("Error closing http stream %s\n", err)
```
Comment (rksharma95) [inline src/cluster/clusterMgmtHandler.go]: I'm not sure if log at error level is needed here. ig it's recoverable in subsequent requests. imo we can use warn here, WDYT?

Comment (Prateeknandle) [reply src/cluster/clusterMgmtHandler.go]: yes ig you're right, will update

**Review by rksharma95 — APPROVED** [review]
LGTM :+1: please squash your commits and it's ready to merge

**Pull request merged**